### PR TITLE
Markov Neural Operators Example

### LINKED ISSUE #15: Implementation of Markov Neural Operators
> Ref. [Markov Neural Operators for Learning Chaotic Systems](https://arxiv.org/abs/2106.06898)

### PR DESCRIPTION
**Pull request by foldfelis**

### CODE REVIEW

**Comment by yuehhua** [comment]
~~This is not a Markov Neural Operators.~~

Now I recognize it is a Markov Neural Operator.

File: example/DoublePendulum/test/data.jl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@testset "get double pendulum chaotic data" begin
```
Comment (yuehhua) [inline example/DoublePendulum/test/data.jl]:
```suggestion
@testset "double pendulum" begin
```

File: example/DoublePendulum/src/data.jl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using CSV
+using DataFrames
+
+export get_double_pendulum_chaotic_data
+
+function register_double_pendulum_chaotic()
+    register(DataDep(
+        "DoublePendulumChaotic",
+        """
+        Dataset was generated on the basis of 21 individual runs of a double pendulum.
```
Comment (yuehhua) [inline example/DoublePendulum/src/data.jl]:
```suggestion
        This dataset is generated based on 21 individual runs of a double pendulum.
```

**Comment by foldfelis** [comment]
> This is not a Markov Neural Operators.

Please take **closer** look at the model introduced by the [paper](https://arxiv.org/abs/2106.06898). With MNO, it can take data of 1 time step and inference the next.

File: README.md [hunk 3]
```diff
@@ -68,25 +68,25 @@ PDE training examples are provided in `example` folder.
...
+julia> using DoublePendulum; DoublePendulum.train()
+```
 
 ## Roadmap
 
 - [x] `FourierOperator` layer
 - [x] One-dimensional Burgers' equation example
-- [ ] Two-dimensional Darcy flow equation example
-- [ ] Two-dimensional Navier-Stokes equation example
+- [ ] `Markov Neural Operators`
```
Comment (yuehhua) [inline README.md]:
```suggestion
- [ ] `MarkovOperator` layer
```

File: README.md [hunk 4]
```diff
@@ -68,25 +68,25 @@ PDE training examples are provided in `example` folder.
...
+- [x] Double Pendulum prediction example
```
Comment (yuehhua) [inline README.md]:
```suggestion
- [x] Double pendulum prediction example
```

File: example/DoublePendulum/src/data.jl (hunk 2 above)
Comment (foldfelis) [reply example/DoublePendulum/src/data.jl]: This meta was copy from home page of dataset.

**Comment by foldfelis** [comment]
Finally, the work was done. 😿

In this PR, I demonstrate the MNO and inferenced though 20+ time steps. The result is updated on the README. Please take a look.

Another, this PR is too large and contains too many trail and error. I'll  squash merge this first. Feel free to open issue and PR if one find it necessary.

**Pull request merged**